DEPENDENCIES.rst

```diff
@@ -114,27 +114,56 @@ and (optionally):
 #. libblas-dev
 #. liblapack-dev
 #. libgoogle-perftools-dev
-#. python3
-#. python3-dev
-#. python3-tables
-#. python3-pandas
-#. python3-numpy
-#. python3-nose
-#. python3-jinja2
-#. cython3
+#. python2-dev    or  python3-dev
+#. python2-tables or  python3-tables
+#. python2-pandas or  python3-pandas
+#. python2-numpy  or  python3-numpy
+#. python2-nose   or  python3-nose
+#. python2-jinja2 or  python3-jinja2
```

Comment:
I don't think that the package name for the v2 packages has the "2".  Only the v3 has the three